packages/api-explorer/src/ApiExplorer.tsx

```diff
@@ -141,6 +157,14 @@ export const ApiExplorer: FC<ApiExplorerProps> = ({
           sdk: alias === allAlias ? null : alias,
         })
       }
+
```

Comment:
This gives code smell, as I can see it continually expanding as new parameters are added. I'm wondering if there is any insight on how to go about syncing multiple parameters at once in this initialization effect.

Reply:
I think a good way to do this will be to augment `initSettings` so that url parameters can be passed in and override persisted settings when applicable. As a result, this effect will then become a one liner that syncs the url with the store using `navigate`.